PublishedModules/glueckself/mpdclient/mpdclient.py

```diff
@@ -0,0 +1,80 @@
+import mpdhelper
+
+from core.base.model.Intent import Intent
+from core.base.model.Module import Module
+from core.dialog.model.DialogSession import DialogSession
+from core.commons.commons import online
+
+class mpdclient(Module):
+	"""
+	Author: glueckself
+	Description: Control an MPD server
+	"""
+
+	_INTENT_PLAY = Intent('Play')
+	_INTENT_STOP = Intent('Stop')
+	_INTENT_NEXT = Intent('Next')
+	_INTENT_PREV = Intent('Prev')
+
```

Comment:
You might want a bit more specific intent names, because names like Play are pretty likely to clash with intent names other modules might want to use aswell

Reply:
You're right, thank you. I think it should be enough to prefix them with mpd, right?

Reply:
yeah that should be absolutely fine